lib/hooks/helpers/index.js

```diff
@@ -148,12 +148,7 @@ module.exports = function(sails) {
       // > NOTE: This is NOT `sails.config.helpers`-- this is `sails.helpers`!
       // > (As for sails.config.helpers, it's set automatically based on our `defaults above)
       sails.helpers = {};
-      Object.defineProperty(sails.helpers, 'inspect', {
-        enumerable: false,
-        configurable: false,
-        writable: true,
-        value: function inspect(){
-
+      sails.helpers[Symbol.for('nodejs.util.inspect.custom')] = function customInspect() {
```

Comment:
Want to keep it non-enumerable so that code looping over the helpers doesn't break and think there's a helper called whatever this new symbol